webapp/news/views.py

```diff
@@ -10,12 +12,21 @@ def index():
     """ Функция, отвечающая за главную страницу. Передает в функцию
             рендеринга макет главной страницы, информацию по погоде """
     title = 'Главная страница'
+    news = News.query.all()
```

Comment:
```suggestion
    news = News.query.order_by(News.published.desc()).limit(10).all()
```
но чтобы это работало, надо типы колонок наследовать от flask_sqlalchemy.SQLAlchemy
```
published = Column(db.DateTime, nullable=False, default=datetime.utcnow)
```